website_livechat_firstname/views/website_livechat_firstname_view.xml

```diff
@@ -0,0 +1,109 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright <YEAR(S)> <AUTHOR(S)>
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+
+<odoo>
+    <data>
```

Comment:
Can remove the data tag. Odoo tag is good enough as long as you don't need some data as noupdate and some as not